khelo/lib/ui/flow/matches/match_detail/components/commentary_over_overview.dart

```diff
@@ -118,27 +120,30 @@ class CommentaryOverOverview extends StatelessWidget {
     BuildContext context,
     BatsmanSummary batsmanSummary,
   ) {
-    return Row(
-      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-      children: [
-        Flexible(
-            child: Text(
-          batsmanSummary.player.name,
-          style: AppTextStyle.subtitle2
-              .copyWith(color: context.colorScheme.textPrimary),
-        )),
-        Text.rich(TextSpan(
-            text: batsmanSummary.runs.toString(),
+    return OnTapScale(
+      onTap: () => UserDetailSheet.show(context, batsmanSummary.player),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          Flexible(
+              child: Text(
+            batsmanSummary.player.name ?? '',
             style: AppTextStyle.subtitle2
                 .copyWith(color: context.colorScheme.textPrimary),
```

Comment:
I think you need to give an onTap action to the player name instead of the whole row. Same as everywhere.